subset/network/test_network

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+sh ./test_dhcplong_send
+sh ./test_ntpupdat_send
```

Comment:
I don't understand why you have all these helper functions if all they do is call one line of python?  Why not just include the python call directly?

Also the "sh" is redundant at the beginning, since they all have #!/bin/bash at the start of them.